tests/features/authentication.feature

```diff
@@ -0,0 +1,13 @@
+Feature: User Authentication
+
+    Scenario: New users
+        Given an anonymous user
+        When I sign up with a valid email and password
+        Then I should be logged in
+        And I should be able to log out
+
+    Scenario: Existing users
+        Given an existing user
+        When I sign up with a valid email and password
+        Then I should be logged in
+        And I should be able to log out
```

Comment:
A feature is described as a BDD use-case/scenario. Tests are run against the feature spec

I think we could start with a few features (authentication, authorization, storage, apis, etc), then refactor later if it becomes unwieldy